frame-metadata/src/v16.rs

```diff
@@ -611,8 +614,10 @@ impl IntoPortable for EnumDeprecationInfo {
 )]
 pub enum VariantDeprecationInfo<T: Form = MetaForm> {
```

Comment:
Could you add a dev note ie:

```rust
// Dev note: codec indexes are added here so that this enum aligns with ItemDeprecationInfo 
```

That way we'll remember why in the future :)